[DOC] Stratio docs Update (#560)
Stratio/kind: stratio-docs/en/modules/ROOT/pages/installation.adoc

+
For automated provisioning in AWS/EKS, it is necessary to execute actions on various AWS services such as EC2, ECR, EKS, Elastic Load Balancing (ELB), etc. While the use or lack of use of these actions will depend on the type of installation, the provider validates that the indicated user has these permissions to run normally.
+

> ** xref:attachment$stratio-aws-unmanaged-policy.json[Descargar permisos permanentes para AWS no gestionado].

Reviewer: ```suggestion
** xref:attachment$stratio-aws-unmanaged-policy.json[Download permanent permissions for AWS unmanaged].
```

> ** xref:attachment$stratio-eks-policy.json[Descargar permisos permanentes para EKS].

Reviewer: ```suggestion
** xref:attachment$stratio-eks-policy.json[Download permanent permissions for EKS].
```

+
** xref:attachment$stratio-gcp-permissions.list[Download permissions for GCP].

> In the case of GKE, in addition to the above permissions, additional permissions will be required for the creation of _clusters_.

Reviewer: ```suggestion
In the case of GKE, in addition to the above permissions, additional permissions will be required to create clusters.
```
